journal/original/2-journal.adoc

```diff
@@ -7742,6 +7775,9 @@ and by whom it is believed, these pages will be read with interest and instructi
 [.signed-section-context-close]
```

Comment:
The section above here, treat the same was as I commented in the modernized, embedding it without a type.